game/src/Game/GameStart.gd

```diff
@@ -67,23 +72,40 @@ func _load_compatibility_mode() -> void:
 			# and if not also search for a Steam install
 			actual_base_path = GameSingleton.search_for_game_path("..")
 		if not actual_base_path:
-			OS.alert(tr("ERROR_ASSET_PATH_NOT_FOUND_MESSAGE"), tr("ERROR_ASSET_PATH_NOT_FOUND"))
-			get_tree().quit()
-			return
+			get_tree().paused = true
+			vic2_dir_dialog.popup_centered_ratio()
+			# Remove with https://github.com/godotengine/godot/pull/81178
+			vic2_dir_dialog.ok_button_text = "VIC2_DIR_DIALOG_SELECT"
+			# WHY WON'T CANCEL AUTO-TRANSLATE WORK NOW?!?!?!?
+			var cancel_button := vic2_dir_dialog.get_cancel_button()
+			cancel_button.auto_translate = false
+			cancel_button.auto_translate = true
+			var failure_func := func() -> void:
+				get_window().mode = Window.MODE_WINDOWED
+				OS.alert(tr("ERROR_ASSET_PATH_NOT_FOUND_MESSAGE"), tr("ERROR_ASSET_PATH_NOT_FOUND"))
+				get_tree().quit()
+			vic2_dir_dialog.canceled.connect(failure_func)
+			await vic2_dir_dialog.dir_selected
+			get_tree().paused = false
+			actual_base_path = GameSingleton.search_for_game_path(vic2_dir_dialog.current_path)
+			if not actual_base_path:
+				failure_func.call()
+				return
 
 	if not _settings_base_path:
 		_settings_base_path = actual_base_path
 		# Save the path found in the search
 		Events.Options.save_settings_to_file()
```

Comment:
The settings base path is only set if it's empty, after that the user can't change it without editing/deleting the settings file. I don't know if command line arg paths should change it (at least after it's already set to something), as those are more likely one-off test things, but as **future additions**, I think the settings path should be changed:
- if the current path is invalid, e.g. if the Vic2 install has been moved. Currently we just accept the settings path with no validation (since we assume it has previously been validated), but a separate `validate_base_path` (which can be used by `search_for_base_path`) would be useful to checking the settings path is still valid and that base paths passed using `--base-path` are valid, so we can update the settings base path if it's no longer valid.
- we could have the settings base path listed in the options menu, editable using the interface added in this PR.

Reply:
This wasn't part of this PR anyway so yeah whatever.